### PR TITLE
Feat : Added Badge

### PR DESCRIPTION
**Pull request by srejitk**
This feature push adds Badges to the Library.

These are the variants added-

1. Round Badge
2. Badge On Icon
3. Tag Badge
4. Avatar Badge

Badge-1
Badge-2
Badge3

### CODE REVIEW

File: stylesheets/components/Badge/Badges.css [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+      background-color: var(--primary-color);
+      color: var(--primary-text);
+  }
+
+  /* Tag Badge Small */
+  .badge--tag--sm {
+      top: -10px;
+      right: -9px;
+      border-radius: 6px;
+      text-decoration: uppercase;
```
Comment (coldpigli) [inline stylesheets/components/Badge/Badges.css]: Does text-decoration has this property?

**Review by coldpigli — COMMENTED** [review]
Left a small doubt regarding text-decoration values.

**Review by SimranjitKaurMaan — APPROVED** [review]

File: stylesheets/components/Badge/Badges.css (hunk 1 above)
Comment (SimranjitKaurMaan) [reply stylesheets/components/Badge/Badges.css]: +1

File: stylesheets/components/Badge/Badges.css [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+      border-radius: 6px;
+  }
+
+  /* Badge on Icon */
+  .badge--icon {
+      top: -10px;
+      right: -10px;
+  }
+
+  /* Rounded Badge */
```
Comment (Sameony) [inline stylesheets/components/Badge/Badges.css]: you can remove such comments as the name of the class is descriptive enough.

File: stylesheets/components/Badge/Badges.css (hunk 1 above)
Comment (Sameony) [reply stylesheets/components/Badge/Badges.css]: text transform*
text decoration is used to add over/underlines to the text .

**Review by Sameony — COMMENTED** [review]
Great code, the class names are very clear and descriptive.

File: stylesheets/components/Badge/Badges.css (hunk 2 above)
Comment (srejitk) [reply stylesheets/components/Badge/Badges.css]: Thanks Sameer. Have removed the additional comments. 
Will keep this in mind from next time

File: stylesheets/components/Badge/Badges.css (hunk 1 above)
Comment (srejitk) [reply stylesheets/components/Badge/Badges.css]: My Bad. It is Text Transform. Thank You  @coldpigli , @SimranjitKaurMaan , @Sameony . It is corrected in the latest commit.

**Review by Sameony — APPROVED** [review]

**Pull request merged**